src/checkers/inference/InferenceQualifierHierarchy.java

```diff
@@ -28,40 +28,24 @@
  */
 public class InferenceQualifierHierarchy extends MultiGraphQualifierHierarchy {
     private final InferenceMain inferenceMain = InferenceMain.getInstance();
-    private final AnnotationMirror varAnnot;
 
     private final SlotManager slotMgr;
     private final ConstraintManager constraintMgr;
 
+    /** Corresponding VarAnnot for the real top qualifier */
```

Comment:
Unmodifiable singleton set containing the VarAnnot corresponding to the real top qualifier.